tgui/packages/tgui/styles/interfaces/GasCanister.scss

```diff
@@ -0,0 +1,11 @@
+.gas-canister-detonator-utility__list-item {
+  line-height: 1.7em;
+}
+
+.gas-canister-detonator-utility__attachment-item {
+  text-transform: capitalize;
+}
+
+.port-atmos-pressure-bar__text {
```

Comment:
While all CSS stuff gets rammed together into one file, I'd prefer if this was pulled into a `PortableAtmos.scss` file, as this is affecting that common interface rather than only your gas canister interface.

Reply:
good idea